app/build.gradle

```diff
@@ -88,7 +89,7 @@ android.applicationVariants.all { variant ->
     def buildType = variant.buildType.name
 
     if (buildType == "release" || buildType == "nightly") {
```

Comment:
I missed this line before: We can remove "nightly" here. The reference browser has only two build types: "release" and "debug".